### ISSUE TITLE
Extend the readme

### ISSUE THREAD

**Issue by lukasoppermann** [GrahamCampbell/Laravel-Binput#28]
Hey @GrahamCampbell I actually am not 100% sure what this does, and how it does what it does. 😄 Could you maybe add a section to explain what it does and also another one with actual usage examples?

From the tests I gathered that it is input sanitisation, but to what extend, and what is actually possible with it? Some more real code examples would be great, thanks.

**Comment by GrahamCampbell** [comment]
Just replace Input:: with Binput::. It's the same.

**Comment by lukasoppermann** [comment]
Ahh, so no options, available?

So I guess `Just replace Input:: with Binput::. It's the same.` Plus one or two sentences about what happens if I do this would be enough to explain it. 😄

**Comment by GrahamCampbell** [comment]
Well, I wrote docs which say how to use it?

**Comment by GrahamCampbell** [comment]
https://github.com/GrahamCampbell/Laravel-Binput#usage

**Comment by GrahamCampbell** [comment]
There are code examples in https://github.com/GrahamCampbell/Laravel-Binput/blob/master/tests/Functional/BinputTest.php.

**Comment by lukasoppermann** [comment]
Yes, this is where I looked and figured out what it does. However as mentioned in #29 I think it would be nice to make it much easier for the user. As stated in the PR, I am happy to add an additional PR to clarify more. I might be a good candidate as I did not use the package before, so for me everything is not as "obvious" as for you. So I might have a better insight into what people who lack the knowledge of what goes on in the packages would actually need (infos). 😄
